pallets/phala/src/mining.rs

```diff
@@ -449,7 +456,7 @@ pub mod pallet {
 		) -> DispatchResult {
 			if let MessageOrigin::Worker(worker) = message.sender {
 				match message.payload {
-					MiningReportEvent::Heartbeat { iterations, .. } => {
+					MiningReportEvent::Heartbeat { iterations, challenge_time, .. } => {
 						// Handle with great care!
```

Comment:
Sorry, The unit of challenge_time is millisesond.

Reply:
Ah, you are right. Fixed.